src/modules/report/base.py

```diff
@@ -22,13 +23,41 @@ def get_services(self):
         services_lock.release()
         return services_data
 
+    def get_severity(self, category):
+        severity = {
+            "Information Disclosure": "medium",
```

Comment:
Are we sure that all information disclosure is medium? @mcherny wdyt?

Reply:
I think that in general severity is by vulnerability and by category. One of my other comments I sent @nshauli is that I don't think severity mapping should be part of reporter.

Reply:
I agree with Micha, if there is a one-to-one mapping between category and severity then this is not adding any information. 

In fact, I think it's worse than that. Isn't this going to return a "low" severity for critical CVE-2018-1002105 which has a category of Privilege Escalation.